src/common/iop_profile.c

```diff
@@ -851,13 +851,29 @@ dt_ioppr_set_pipe_output_profile_info(struct dt_develop_t *dev,
   return profile_info;
 }
 
-dt_iop_order_iccprofile_info_t *dt_ioppr_get_histogram_profile_info(struct dt_develop_t *dev)
+static void _dt_ioppr_get_histogram_profile_type(dt_colorspaces_color_profile_type_t *profile_type,
```

Comment:
static internal function should not start with "_dt" just "_ioppr_...". I know there is some wrong naming but they should be eliminated. That's just a convention, right.

Reply:
Alright, thanks for noticing this! On the other hand, could it just be `_get_histogram_profile_info` (as the `ioppr_` part would also have been a "namespace" but this function is internal)?

Reply:
Yes, fine by me and probably better indeed as we generally remove this in other places.